auth.go

```diff
@@ -310,6 +310,81 @@ func CreateContractInvokeRequestWithMessage(
 	}
 }
 
+func verifyAuthRequest(request protocol.AuthorizationRequestMessage) error {
```

Comment:
maybe it should be public function? so users, can verify that they created a valid request?

Reply:
made public